### ISSUE TITLE
Avoid showing content of password file on copying (-c)

### ISSUE THREAD

**Issue by shellowl** [gopasspw/gopass#1323]
### Summary
Since version 1.9, `gopass -c` outputs the password in cleartext, the comment, the URL and the username.
This wasn't the standard behavior before.

### Steps To Reproduce
`gopass -c <string>`

### Expected behavior
`gopass -c` shouldn't show any content of the password file by default.

### Environment
- OS: Arch Linux 5.6.8-1
- gopass Version: 1.9.0-git+HEAD (2020-05-03 20:29:17) go1.14.2 linux amd64
- Installation method: Arch community repo

**Comment by doronbehar** [comment]
Also, `gopass show -f` tries to put the password in the clipboard unconditionally, and if that fails (if not in an X session) the password isn't shown at all. This change broke for me some startup scripts which put passwords in environmental variables on shell startup.

**Comment by doronbehar** [comment]
Not only that, even if `gopass show -f` runs in an Xsession and it's piped to an external command or is used with:

```
export MY_PASSWD=$(gopass show -f my/password)
```

The text:

```
✔ Copied software/common/mpd to clipboard. Will clear in 45 seconds.
```

Appears in the env var as well.

**Comment by quidome** [comment]
for me, also on arch, -c doesn't even copy anything to clipboard. Looks like -c is ignored or something as the password is printed to the screen.

**Comment by doronbehar** [comment]
@quidome Maybe having `xsel` installed would help?

**Comment by dominikschulz** [comment]
Can someone verify this is also broken at or after 05979a825ab10ca014052d55194273a43808b9f4 ?

This would help triaging. I'll try to fix this tonight, if I can reproduce it.

**Comment by doronbehar** [comment]
Everything seems to act the same, at current master.

**Comment by koalp** [comment]
> @quidome Maybe having `xsel` installed would help?

I have the same issue than quidome (on arch, no password copied, password shown on screen), and have xsel installed.

**Comment by quidome** [comment]
> @quidome Maybe having `xsel` installed would help?

Didn't have it installed (neither in the past) and installed it to test, did not make a difference.

**Comment by oscarpax** [comment]
TL;DR
I can confirm problem solved at https://github.com/gopasspw/gopass/commit/5e4dcc93d4923b85d8f387e7706f0aec09bddd2f on Arch
....
At https://github.com/gopasspw/gopass/commit/5e4dcc93d4923b85d8f387e7706f0aec09bddd2f 
`gopass <entry>` 
`gopass show <entry>` 
`gopass  -c <entry>`
`gopass -C <entry>`
all result with password copied to clipboard and no content shown

Only
`gopass show -f <entry>`
copies password to clipboard and outputs content

**Comment by dominikschulz** [comment]
So at `5e4dcc9` this looks almost OK:

Testing with `autoclip` `false`:
```
$ ./gopass config autoclip false
autoclip: false
$ ./gopass foo/bar
z4vbViW110qbj66h7wDqM8VB
second
third
$ ./gopass -c foo/bar
✔ Copied foo/bar to clipboard. Will clear in 45 seconds.
$ ./gopass -C foo/bar
✔ Copied foo/bar to clipboard. Will clear in 45 seconds.
z4vbViW110qbj66h7wDqM8VB
second
third
$ ./gopass show foo/bar
z4vbViW110qbj66h7wDqM8VB
second
third
$ ./gopass show -c foo/bar
✔ Copied foo/bar to clipboard. Will clear in 45 seconds.
$ ./gopass show -C foo/bar
✔ Copied foo/bar to clipboard. Will clear in 45 seconds.
z4vbViW110qbj66h7wDqM8VB
second
third
```
So it seems with `autoclip` `false` everything works as expected / documented.


Testing with `autoclip` `true`:
```
$ ./gopass config autoclip true
autoclip: true
$ ./gopass foo/bar
✔ Copied foo/bar to clipboard. Will clear in 45 seconds.
$ ./gopass -c foo/bar
✔ Copied foo/bar to clipboard. Will clear in 45 seconds.
$ ./gopass -C foo/bar
✔ Copied foo/bar to clipboard. Will clear in 45 seconds.
$ ./gopass show foo/bar
✔ Copied foo/bar to clipboard. Will clear in 45 seconds.
$ ./gopass show -c foo/bar
✔ Copied foo/bar to clipboard. Will clear in 45 seconds.
$ ./gopass show -C foo/bar
✔ Copied foo/bar to clipboard. Will clear in 45 seconds.
# Bug. -C should copy AND print the content.
$ ./gopass show -f foo/bar
✔ Copied foo/bar to clipboard. Will clear in 45 seconds.
z4vbViW110qbj66h7wDqM8VB
second
third
```

However with `autoclip` `true` setting `-C` will only copy, not print. This should be fixed.
